public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -114,7 +115,11 @@ export class PrometheusDatasource {
     }
 
     var allQueryPromise = _.map(queries, query => {
-      return this.performTimeSeriesQuery(query, start, end);
+      if (query.format === 'time_series') {
```

Comment:
The Prometheus terms are "range query" vs. "instant query" - it might make sense to follow that terminology here for consistency. Both return data for time series, but the instant query just returns a single point for each "series" :)

Reply:
Oops, sorry for inconsistence.
Finally, I revert this line, and change to use checkbox, property name is `instant`.

Reply:
@juliusv I agree. It makes sense to use the same vocabulary as much as possible.

@mtanda great! please use the gf-form-switch component for checkboxes. Ex https://github.com/grafana/grafana/blob/master/public/app/plugins/panel/graph/tab_legend.html#L26